dag/artificial_intelligence.yml

```diff
@@ -221,4 +221,14 @@ steps:
   data://garden/us_census_bureau/2025-11-17/datacenter_construction:
     - data://meadow/us_census_bureau/2025-11-17/datacenter_construction
   data://grapher/us_census_bureau/2025-11-17/datacenter_construction:
-    - data://garden/us_census_bureau/2025-11-17/datacenter_construction
\ No newline at end of file
+    - data://garden/us_census_bureau/2025-11-17/datacenter_construction
+
+
+  # IEA Energy Consumption by AI dataset
+  data://meadow/artificial_intelligence/2025-11-07/energy_ai_iea:
+    - snapshot://artificial_intelligence/2025-11-07/energy_ai_iea.xlsx
+  data://garden/artificial_intelligence/2025-11-07/energy_ai_iea:
+    - data://meadow/artificial_intelligence/2025-11-07/energy_ai_iea
+    - data://garden/energy/2025-06-27/electricity_mix
+  data://grapher/artificial_intelligence/2025-11-07/energy_ai_iea:
```

Comment:
Unless you are planning to have multiple energy_ai_* datasets, I think it would be preferrable to simply call the steps `energy_ai` (or `energy_and_ai`, to match the title of the original dataset). That way we avoid hardcoding the producer (in fact, since the electricity mix is included, IEA is no longer the only origin). But if the refactor is too inconvenient, you can leave it this way.